src/components/configuration/ConfigurationView.css

```diff
@@ -5,6 +5,11 @@
   .configurationView .icon {
     width: 30px;
     height: 30px; }
+  .configurationView .list-group-item-container {
```

Comment:
@dlabaj for some reason this isn't in the codebase but I think its rendered some how. Whats the best way to remove it?

Reply:
If you run `npm run build-css` it should remove this block from the file.

Reply:
@craicoverflow yeah I only added it to the `.scss` file, but when I run `npm run build-css` it removes it from the `css`... thanks for that